src/main/java/dude/command/Command.java

```diff
@@ -5,6 +5,10 @@
 import dude.util.TaskList;
 import dude.util.Ui;
 
+/**
+ * The abstract task contains the action of the command and whether the command calls for an exit.
+ */
+
```

Comment:
Should you remove this empty line between class header comment and the class itself?